opengl.sh

```diff
@@ -6,6 +6,9 @@ system_requirement_missing: |
     * On Ubuntu-compatible systems you probably need: libglu1-mesa-dev
 system_requirement: ".*"
 system_requirement_check: |
-  printf "#include <GL/glu.h>\n" | cc -xc - -c -o /dev/null
+  case $ARCHITECTURE in 
```

Comment:
I suggest to do the OS check in the following way:

if [ `uname` = Darwin ]; then
  printf "#include <OpenGL/glu.h>\n" | cc -xc - -c -o /dev/null
else
  printf "#include <GL/glu.h>\n" | cc -xc - -c -o /dev/null
fi

Using above written lines the one wouldn't need to extra specify $ARCHITECTURE variable during the installation

Reply:
I agree, it seems a good idea to me.
Antonio